### ISSUE TITLE
Tests not build anymore on iOS after nativescript-core refactorization

### ISSUE THREAD

**Issue by surdu** [NativeScript/NativeScript#8014]
**Environment**
 - CLI: 6.3.0-2019-10-17-161841-13944 (@next)
 - Cross-platform modules: master

**Describe the bug**
After updated the master with the latest `nativescript-core` refactorization, I was instructed to upgrade CLI to at least 6.2.0. I upgraded it to @next.

After this, whenever I try to run the tests on iOS I get:

```
error: Multiple commands produce '<PbxCp /Users/nicolae.surdu/work/NativeScript/tests/platforms/ios/build/Debug-iphonesimulator
/tests.app/Frameworks/TNSWidgets.framework>':
1) Target 'tests' (project 'tests') has copy command from '/Users/nicolae.surdu/work/NativeScript/tests/node_modules/@nativescr

ipt/core/node_modules/tns-core-modules-widgets/platforms/ios/TNSWidgets.framework' to '/Users/nicolae.surdu/work/NativeScript/t
ests/platforms/ios/build/Debug-iphonesimulator/tests.app/Frameworks/TNSWidgets.framework'
2) Target 'tests' (project 'tests') has copy command from '/Users/nicolae.surdu/work/NativeScript/tests/node_modules/tns-core-m
odules/node_modules/tns-core-modules-widgets/platforms/ios/TNSWidgets.framework' to '/Users/nicolae.surdu/work/NativeScript/tes
ts/platforms/ios/build/Debug-iphonesimulator/tests.app/Frameworks/TNSWidgets.framework'
```

**To Reproduce**
1. have this repo cloned and set-up
2. `npm install -g nativescript@next`
3. Inside this repo: `cd tests & tns run ios`

**Expected behavior**
The test app should run

**Comment by khanyuriy** [comment]
I have same issue. Is there any workarounds?

**Comment by DimitarTachev** [comment]
It seems that you have two instances of `tns-core-modules-widgets` in the app. This package is a dependency of `@nativescript/core` and should not be duplicated.

The `@nativescript/core` and `tns-core-modules` (just a proxy to `@nativescript/core`) packages a meant to be used with exactly the same version. (e.g. 6.3.1). In this way, the `@nativescript/core` package will always be deduped and the `tns-core-modules-widgets` duplication will not be possible.

In other words, you have to use the same version of `@nativescript/core` and `tns-core-modules` and run `npm dedupe` if needed. You could check the problematic dependency tree by executing `npm ls tns-core-modules-widgets`.
